### DISCUSSION TITLE
UM_TinyPico stubs installation

### DISCUSSION THREAD

**Discussion by G234i** [Josverl/micropython-stubs#636] [Q&A] [answered]
So I am new into microcontrollers and micropython, but I have a UM_TinyPico board I am trying to mess around with.
I  am using VS Code and I managed to get the "core" and "Generic" stubs to work using `pip install micropython-esp32-stubs`.

However I am missing some UM_TinyPico specific imports (as those are not included after running the `pip install` install command?).
I tried passing micropython-esp32-<board>-stubs as an argument to the `pip install` command (of course replacing <board>)
but something leads me to believe that  that is not intended to work like that.

Is there another way to get those board specific stubs other than cloning the repo by hand to the project folder/using symbolic links?

**Comment by Josverl** [comment, marked as answer]
Sorry for not replying earlier, I think I need to set a notification for these discussions.

currently I had not (yet) published more specific stubs such as the `UM_TinyPico`, but I can do that 
then the install would be something like:  `pip install -U micropython-esp32-um_tinypico-stubs`

I have now (manually) published the v1.19.1 stubs. and I'll add that to the automated publishing process 
I
